spec/models/cart_product_spec.rb

```diff
@@ -0,0 +1,132 @@
+RSpec.describe CartProduct, type: :model do
```

Comment:
explore the usage of the [annotate](https://github.com/ctran/annotate_models) gem. It could be useful when you want to know the column names easily